### DISCUSSION TITLE
Adding UV?

### DISCUSSION THREAD

**Discussion by abe-101** [browniebroke/pypackage-template#844] [Q&A] [answered]
Any interest in replacing poetry with UV for the template?
Or adding it as a optional feature in the generation options?

**Comment by browniebroke** [comment]
Yes I intend to move to uv, no need to worry about making it an option.

I wanted to migrate a few of my packages manually, let it run for a bit and learn any limitations with the ecosystem. Then migrate this template

I started a PR on a small package I own https://github.com/browniebroke/netlify-builds/pull/770 but life got in the way 😉

I'm hoping to pick this up in early 2025

**Reply by browniebroke** [reply, marked as answer]
Done in https://github.com/browniebroke/pypackage-template/pull/847

**Reply by abe-101** [reply]
Thanks!
